src/components/Game.js

```diff
@@ -1,19 +1,19 @@
-import { useState, useEffect } from "react";
```

Comment:
Can we be more consistent and do import in the same way across the app?

Reply:
Agreed, we are taking two different approaches. We should discuss.